### ISSUE TITLE
Rescanning Blocks Graphical Display Error of Hush Logo

### ISSUE THREAD

**Issue by oDinZu** [DenioD/SilentDragonLite-1#102]
**Operating System:** Ubuntu 20.04
**Application Version:** v1.3.1
**Theme:** Midnight
**Note:** Running application in CLI with STDOUT

After clicking rescan via file menu options, I seen the blocks being rescanned on initial GIF. Next, I closed the GIF rescan by clicking the X button top right. Furthermore, I click X button on core application, then another SDL logo pops up similar to how the GIF does. Now, this popup dialog box, has a graining SDL logo and is out of alignment. This screen has no loading details and is strictly 'static'. See images below.
![Screenshot from 2020-06-05 21-53-34](https://user-images.githubusercontent.com/18726788/83933600-16e14e80-a778-11ea-8272-d7db3fd5b5de.png)

To further go, I exited out of this by also clicking the X button, then I clicked the X button on core application again and the application closed, although STDOUT via CLI displayed hushd still scanning for blocks.

![Screenshot from 2020-06-05 22-04-28](https://user-images.githubusercontent.com/18726788/83933669-aab31a80-a778-11ea-9356-f0d4b0f4b71f.png)

**Comment by oDinZu** [comment]
The conclude, the Syncing completed, which is good. Although, without the CLI and STDOUT, I would have tried opening the app again and probably cause an error.

Update: On relaunch of the app, it is syncing from place closed the app. Even CLI stated it was finished, but it must not have been or another variable saved last state of the block index and continued to stay open via CLI to complete syncing.

**Comment by oDinZu** [comment]
It is syncing again and the app info is saying it doesn't have a connection..again, the application is still syncing; to sync, we need nodes and if we have nodes, then we have connection.

![Screenshot from 2020-06-05 22-17-32](https://user-images.githubusercontent.com/18726788/83933885-89ebc480-a77a-11ea-8cb9-348eed4d8fc1.png)

**Comment by leto** [comment]
@csharpee can you summarize what you think the "bug" is, in one sentence? I am not sure I understand

**Comment by oDinZu** [comment]
Do you see the logo on the first screenshot? 

That logo appears on midnight theme and is grainy + out of alignment. This happens when I do the following below. 

Rescan the blocks then exit out of dialog GIF box, then exit out of application, then **that screen appears**

**Comment by oDinZu** [comment]
To add a bookmark, this bug begun after encryption was added. Hopefully this aids in an area of reference.
